lua/swenv/api.lua

```diff
@@ -14,7 +14,12 @@ local ORIGINAL_PATH = vim.fn.getenv('PATH')
 local current_venv = nil
 
 local update_path = function(path)
-  vim.fn.setenv('PATH', path .. '/bin' .. ':' .. ORIGINAL_PATH)
+  local os = string.lower(vim.loop.os_uname().sysname)
+  if string.find(os, 'windows') then
+    vim.fn.setenv('PATH', path .. '/Scripts' .. ';' .. ORIGINAL_PATH)
+  else
+    vim.fn.setenv('PATH', path .. '/bin' .. ':' .. ORIGINAL_PATH)
+  end
```

Comment:
```suggestion
  local os = vim.loop.os_uname().sysname
  if os == 'Windows_NT' then
    vim.fn.setenv('PATH', path .. '\\Scripts' .. ';' .. ORIGINAL_PATH)
  else
    vim.fn.setenv('PATH', path .. '\\bin' .. ';' .. ORIGINAL_PATH)
  end
```

Reply:
Shouldn't this be:
```lua
if is_windows() then
  vim.fn.setenv('PATH', path .. '\\Scripts' .. ';' .. ORIGINAL_PATH)
else
  vim.fn.setenv('PATH', path .. '/bin' .. ':' .. ORIGINAL_PATH)
end
```
? Ie use backslash and `;` for windows and otherwise forward slash and `:` for linux/mac.

Also the original version selects for any string containing `windows` but here equality with `Windows_NT` is used, do you know if that can vary between windows versions?

From docs it also seems `vim.uv.os_uname` should be used instead of `vim.loop.os_uname`?

Reply:
I didn't notice the colon for Linux/Mac and windows accepts forward slash too. 

For `Windows_NT`, I doubt it varies between Windows versions because the Neovim [Makefile, L1-12](https://github.com/neovim/neovim/blob/master/Makefile#L1-L12) runs a check against that string for Windows.

Yes you are right here. `vim.uv.os_uname` is the right choice

Reply:
Merged this in #48